tests/std/tests/P1502R1_standard_library_header_units/custom_format.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from pathlib import Path
+
+from stl.test.format import STLTestFormat, TestStep
+
+
+class CustomTestFormat(STLTestFormat):
+    def getBuildSteps(self, test, lit_config, shared):
+        shared.exec_dir = test.getExecDir()
+        output_base = test.getOutputBaseName()
+        output_dir = test.getOutputDir()
+        source_path = Path(test.getSourcePath())
+
+        stl_headers = [

```

Comment:
do we need to remember to remove this code when upstream learns how to module? I suppose with lit it's not super likely since they will be concerned about clang.

Reply:
I believe that this code is permanent, as we need to craft custom command lines to build these header units (and `lit` never tries to act like a build system).